dojo/tools/cargo_audit/parser.py

```diff
@@ -29,16 +29,20 @@ def get_findings(self, filename, test):
                 else:
                     categories = ''
                 description = categories + f"\n**Description:** `{advisory.get('description')}`"
-                if "functions" in item["affected"]:
-                    affected_func = [f'{func}: {", ".join(versions)}'
-                                     for func, versions in item["affected"]["functions"].items()]
-                    description += f"\n**Affected functions**: {', '.join(affected_func)}"
-                references = f"{advisory.get('url')}\n" + '\n'.join(advisory['references'])
-                date = advisory.get('date')
+
+                if item["affected"] is not None:
+                    if "functions" in item["affected"]:
```

Comment:
`if item["affected"] is not None and "functions" in item["affected"]:` would have been more clear.